docs/overview/cmake-can-not.rst

```diff
@@ -13,21 +13,21 @@ What can't be done with CMake
 .. Can't find good reference. Let me know if can find a better link (Ruslo)
 
 :ref:`CMake <CMake>` has it's strengths and weaknesses. Most of the drawbacks
-mentioned here can be workarounded by using approaches that may differs from
-your normal workflow but reach the end goal. Try to look at them from another
-angle, think of the picture in a whole and remember that advantages definitely
+mentioned here can be worked around by using approaches that may differ from
+your normal workflow, yet still reach the end goal. Try to look at them from another
+angle; think of the picture as a whole and remember that the advantages definitely
 outweigh the disadvantages.
 
 Language/syntax
 ===============
 
-This is probably the first thing you will hit with. :ref:`CMake <CMake>`
+This is probably the first thing you will be hit with. The :ref:`CMake <CMake>`
 language is not something you can compare with what you have likely used
 before. There are no classes, no maps, no virtual functions or lambdas. Even
-parsing functions input arguments and returning results is something quite
+parsing functions and input arguments then returning results is something quite
```

Comment:
I mean "input arguments of the different functions", so "and" is not correct here.

Reply:
I don't think "then" is a right word here because you can return result without parsing arguments and you can parse arguments without returning results.

Reply:
I changed the wording based on the first point, but the second point doesn't make sense. Of course you can return without parsing or parse without returning, but what this sentence is doing is providing an example of something that might be tricky for beginners. "Even this task can be tricky".